### DISCUSSION TITLE
After the update this morning unable to get to login page.

### DISCUSSION THREAD

**Discussion by bclinton1** [photoview/photoview#333] [Q&A and Troubleshooting] [answered]
Greetings folks! New user to photoview and it was working yesterday.....after the update this monring I am unable to get to the login page. My log has this entry when I attempt to launch the webui with unraid. Was not sure if it was related to the update or if I have something not configured right. :)

2021/04/20 06:40:08 Did not find auth-token cookie
2021/04/20 06:40:11 [0;34;49mGET [0;34;49m0[0m 192.168.1.24:8694/api/graphql [0;36;49m3.446236865s[0m unauthenticated

**Comment by viktorstrate** [comment, marked as answer]
What happens when you write your credentials into the login page?

**Reply by bclinton1** [reply]
It does not load the login page. I see it flash and then blank screen.

**Reply by viktorstrate** [reply]
Okay, does it work if you log in in Incognito mode?

**Reply by bclinton1** [reply]
It does indeed work with incognito mode. Clearing cookies fixed it! Thanks my friend!
